mobile/build.gradle

```diff
@@ -127,8 +127,7 @@ dependencies {
         transitive = true
     }
 
-    fullImplementation 'com.google.firebase:firebase-core:12.0.1'
-    fullImplementation 'com.google.firebase:firebase-messaging:12.0.1'
+    fullImplementation 'com.google.firebase:firebase-messaging:17.3.4'
```

Comment:
Can you exclude the analytics part? https://github.com/siacs/Conversations/commit/c2067e3b56364317bc384b55e320b72326fb1de6
And add `<meta-data android:name="firebase_analytics_collection_deactivated" android:value="true" />` to the manifest (https://firebase.google.com/support/guides/disable-analytics)?